platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -515,19 +513,6 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
             eventStreamManagerName = String.valueOf(selfId);
```

Comment:
local variables `address` and `eventStreamManagerName` can now be deleted

Reply:
deleted